templates/.github/workflows/go-test.yml

```diff
@@ -10,8 +10,8 @@ jobs:
         go: [ "1.16.x", "1.17.x" ]
     env:
       COVERAGES: ""
-    runs-on: ${{ matrix.os }}-latest
-    name: ${{ matrix.os}} (go ${{ matrix.go }})
+    runs-on: ${{ format('${0}-latest', matrix.os) }}
```

Comment:
was the old syntax wrong? I'm a bit confused because it apparently worked just fine :)

Reply:
It worked but the schema from https://json.schemastore.org/github-workflow.json didn't account for it. I'll see if there's a way to express what we were doing in that schema - if so I'll create a PR but, personally, I'd be fine with the setup proposed here too.

Reply:
I see. Slight preference for keeping what we had, because it was simpler, but I also don't think it matters too much :)

Reply:
SGTM! Thanks for being thorough :)